src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -690,6 +697,16 @@ private static void addConversionConfigs(ConfigDef configDef) {
             Width.SHORT,
             BEHAVIOR_ON_MALFORMED_DOCS_DISPLAY,
             new EnumRecommender<>(BehaviorOnMalformedDoc.class)
+        ).define(
+                    EXTERNAL_VERSION_HEADER_CONFIG,
```

Comment:
nit: This can be formatted for indentation

Reply:
done, thanks